docs/source/conf.py

```diff
@@ -33,7 +33,6 @@
     "sphinx.ext.napoleon",
     "sphinx_tabs.tabs",
     # "nbsphinx",
```

Comment:
Not related to this PR, but what is `nbsphinx` used for?

Reply:
It is an extension that helps render jupyter notebooks as html. We don't have any notebooks in our docs yet so it's commented out.